pkgs/development/web/nodejs/nodejs.nix

```diff
@@ -249,7 +250,13 @@ let
 
     pos = builtins.unsafeGetAttrPos "version" args;
 
-    inherit patches;
+    patches = patches ++ [
+      # fixes test failure
+      (fetchpatch {
+        url = "https://github.com/nodejs/node/commit/b6fe731c55eb4cb9d14042a23e5002ed39b7c8b7.patch";
+        hash = "sha256-y45G9Vc58Nh0I+kSS3iRnoZe7se55rk3PVMpgZgPY+U=";
+      })
```

Comment:
This commit is going to get backported to Node.js release lines and won't apply anymore. Instead, let's add it to `20.nix`, `22.nix`, and `23.nix` (we don't run tests on nodejs_18) so it's easier to update Node.js in the future.

Reply:
Done :+1: